src/modal/modal.js

```diff
@@ -72,6 +72,12 @@ class Modal extends React.Component<ModalPropsT, ModalStateT> {
         Then pass backdrop overrides to DialogContainer instead. Tha will help you with
         the next major version upgrade.`);
       }
+      // $FlowFixMe: flow complains that this prop doesn't exist
+      if (this.props.closable) {
```

Comment:
this should have a `__DEV__` guard

Reply:
it's not visible in this diff, but it's part of a bigger __DEV__ condition: https://github.com/uber/baseweb/blob/4f21b3af2aa8bf793b496d515ede4be1fe36e2b1/src/modal/modal.js#L61-L80